README.md

```diff
@@ -181,32 +178,272 @@ run
 make -s
 ```
 
-The -s is for 'silent'. Successful output looks like this
+The -s is for 'silent'. Successful output looks like below and exists with 0 exit code.
 
 ```
+[user@host:~/workspace/google/terraform-google-container-vm]$ make -s
```

Comment:
Why did you put full log here?

Reply:
> Why did you put full log here?

I highlighted the importance of successful exit code above, so I decided to leave the command call and the exit code checks in.. 
This way it looks better to me, but I'm open for discussion :)

Reply:
I think **$** is sufficient, don't need **user@host:~/workspace/google/terraform-google-container-vm**

Reply:
Sure, fixed! Thank you